### PR TITLE
Fixing Bugs From Testing

### PR DESCRIPTION
**Pull request by vychun**
Some of the bugs that came up that during our testing
- The FAQ updates 
  - seventh -> eighth
  - the new blog post
  - recommended people _dont_ bring resumes
- Meet the Team
  - tongue fix
  - kyle's new fun fact
  - hover/tap for mobile on meet the team
- footer link change
- adding padding to the FAQ title on mobile to prevent it from being covered by the nav when it's clicked on
- made links bold/strong to make them easily readable
- made the about images a little larger on mobile
- prevented wrapping on about items because it would wrap oddly

### CODE REVIEW

File: src/pages/team/index.jsx [hunk 1]
```diff
@@ -302,7 +304,7 @@ const Team = () => {
               className="team__cta"
               role="button"
             >
-              Email us
+              Sign up for our newsletter
```
Comment (vychun) [inline src/pages/team/index.jsx]: actually @ctlnwng This should either say Email us or I can change the link for the button

File: src/data/about-content.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
       "text": "Read the stories",
       "url": "/stories"
     },
-    "content": "<p>Memorable stories from HackBeanpot are just one of the things that make us unique. Our attendees range from graduate CS students to designers who have never written a line of code in their life!</p>"
+    "content": "<p>Memorable stories from HackBeanpot are just one of the things that make us unique. Our attendees range from CS graduate students to beginner designers who have never written a line of code before.</p>"
```
Comment (vychun) [inline src/data/about-content.json]: should this say `beginning` designers? either way is fine though

Comment (ctlnwng) [reply src/data/about-content.json]: Sure I also think either way is fine, your call!

Comment (vychun) [reply src/data/about-content.json]: Oh I already pushed so it's fine

**Pull request merged**